.github/workflows/pr-release-workflow.yml

```diff
@@ -0,0 +1,29 @@
+name: PR Release Workflow
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    types:
+      - milestoned
+      - closed
+
+jobs:
+  backport-commit:
+    runs-on: ubuntu-22.04
```

Comment:
`ubuntu-24.04` here? I want to start migrating things soonish, and it doesn't seem like anything will break for this workflow by bumping it.